docs/project-overview/product-support-levels.md

```diff
@@ -0,0 +1,39 @@
+# Connector Support Levels
```

Comment:
Maybe we should just name this file `connector-support-levels.md`, since it is connector-specific now?